doc/can-define-object.md

```diff
@@ -0,0 +1,479 @@
+@module {function} can-define-object
+@parent can-observables
+@collection can-ecosystem
+@group can-define-object/object.behaviors 0 behaviors
+@group can-define-object/object.static 1 static
+@group can-define-object/object.types 2 types
+@group can-define-object/object.other 3 other
+@alias can.DefineObject
+@templateRender true
+
+@description Create observable objects.
+
+@signature `new DefineObject([props])`
+
+  The `can-define-object` module exports the `DefineObject` class.
+
+  Calling `new DefineObject(props)` creates a new instance of DefineObject or an [can-define-object/object.extend extended] DefineObject. Then, `new DefineObject(props)` assigns every property on `props` to the new instance.  If props are passed that are not defined already, those properties are set on the instance.  If the instance should be sealed, it is sealed.
+
+  ```js
+  import { DefineObject } from "can/ecosystem";
```

Comment:
I think you want to use `can/everything` so it will be rewritten correctly for codepen.